### PR TITLE
(検証用)Sonar解析のGitHubレビュー連携

### PR DESCRIPTION
**Pull request by yuto-suzuki**
検証用ブランチです
検証完了次第削除します

### CODE REVIEW

File: src/main/java/biz/kaipoke/log4j/appender/redis/KaipokeRedisAppender.java [hunk 1]
```diff
@@ -16,6 +15,19 @@
 	private static final Pattern PATTERN_URI = Pattern.compile("(?:URI\\[)(/[\\w/:%#\\$&\\?\\(\\)~\\.=\\+\\-]+)(?:])");
 	private static final Pattern PATTERN_REFERRER = Pattern.compile("(?:REFERER\\[)(https?://[\\w/:%#\\$&\\?\\(\\)~\\.=\\+\\-]+)(?:])");
 
+	@Deprecated
+	protected String fuga() {
+		if (true) {
+			if (true) {
```
Comment (smsbot-rotom) [inline src/main/java/biz/kaipoke/log4j/appender/redis/KaipokeRedisAppender.java]: MAJOR Merge this if statement with the enclosing one. [rule](http://sonarqube.digda.team/coding_rules#rule_key=squid%3AS1066)

File: src/main/java/biz/kaipoke/log4j/appender/redis/KaipokeRedisAppender.java [hunk 2]
```diff
@@ -16,6 +15,19 @@
...
+				if (true) {
```
Comment (smsbot-rotom) [inline src/main/java/biz/kaipoke/log4j/appender/redis/KaipokeRedisAppender.java]: MAJOR Merge this if statement with the enclosing one. [rule](http://sonarqube.digda.team/coding_rules#rule_key=squid%3AS1066)

**Comment by smsbot-rotom** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 2 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by yuto-suzuki** [comment]
レビュー連携検証完了のためクローズ

**Pull request closed without merging**